packages/ra-ui-materialui/src/layout/Title.tsx

```diff
@@ -8,39 +8,41 @@ import { RaRecord, warning } from 'ra-core';
 import { PageTitleConfigurable } from './PageTitleConfigurable';
 
 export const Title = (props: TitleProps) => {
-    const { defaultTitle, title, preferenceKey, ...rest } = props;
-    const [container, setContainer] = useState(() =>
-        typeof document !== 'undefined'
-            ? document.getElementById('react-admin-title')
-            : null
-    );
+  const { defaultTitle, title, preferenceKey, nonconfigurable, ...rest } = props;
+  const [container, setContainer] = useState(() =>
+    typeof document !== 'undefined'
+      ? document.getElementById('react-admin-title')
+      : null
+  );
```

Comment:
Please configure your IDE to use our current Prettier configuration, to avoid breaking the indentation of the file